### ISSUE TITLE
Audit source code

### ISSUE THREAD

**Issue by twzkxkan** [surespot/android#133]
In order to rebuild trust in your software, it should be professionally audited like [TextSecure](https://eprint.iacr.org/2014/904.pdf).

**Comment by BlackADeath** [comment]
I agree. In fact the application is even mentioned in the report ghost linked. The company even mentioned an interest in comparing $This to TextSecure.

**Comment by grrrrr** [comment]
Surespot could be submitted to Mozilla's [Secure Open Source](https://blog.mozilla.org/blog/2016/06/09/help-make-open-source-secure/) fund
